release.sh

```diff
@@ -1,5 +1,95 @@
 #!/bin/sh
```

Comment:
It may not matter so much for how we use it, but since this has some bashisms, we might consider changing this to something like #!/bin/bash or #!/usr/bin/env bash for maximum portability.

Reply:
OK, I changed it to #!/bin/bash